### ISSUE TITLE
convert: "-page" behaviour with "+append"ed gif frame

### ISSUE THREAD

**Issue by lewa** [ImageMagick/ImageMagick#326]
```
# convert -size 150x100 xc:red red.gif
# convert -size 50x100 xc:green green.gif
# convert -size 25x100 xc:blue blue.gif
# convert -delay 100 red.gif -page +40+0 \( green.gif blue.gif +append \) animation.gif
# identify animation.gif
animation.gif[0] GIF 150x100 150x100+0+0
animation.gif[1] GIF 75x100 150x100+60+0
```

The second frame of animation.gif has 60px offset instead of the expected 40px. Is this a counter-intuitive feature or a bug? Presumably, 60px = 40 / 50 * (50 + 25)

ImageMagick 6.7.7-10 2016-11-29 Q16
ImageMagick 7.0.3-10 Q8 x64 2016-12-11

**Comment by mikayla-grace** [comment]
```
convert -delay 100 red.gif \( green.gif blue.gif +append -set page +40+0 \) animation.gif
```

**Comment by lewa** [comment]
"-set page" works like a charm indeed. However, I wonder if -page/+append behaviour is intended and what the logic behind it is.

**Comment by mikayla-grace** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in GIT master branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ http://www.imagemagick.org/download/beta/ by sometime tomorrow.

**Comment by dlemstra** [comment]
Is your issue resolved with the latest version @lewa ?

**Comment by lewa** [comment]
Yes, the issue has been resolved in beta 7.0.4-1~beta20161224; the latest version works like a charm as well. Thank you very much for prompt resolution.

```
# identify animation.gif
animation.gif[0] GIF 150x100 150x100+0+0
animation.gif[1] GIF 75x100 150x100+40+0
```

IImageMagick 7.0.4-6 Q8 x64 2017-01-27
